src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +172,33 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::get_output_element_type(const Napi::CallbackInfo& info) {
+    if(info.Length() != 1 || !info[0].IsNumber()) {
+        reportError(info.Env(), "Invalid argument. Expected a single number for output index.");
+        return info.Env().Undefined();
+    }
+
+    try {
+        auto idx = info[0].As<Napi::Number>().Int32Value();
+        auto output = _model->output(idx);
+        auto element_type = output.get_element_type();
+        std::string type_name = element_type.get_type_name();
+        std::unordered_map<std::string, std::string> type_map = {
+            {"float", "f32"},
+            {"float16", "f16"},
+            {"int32", "i32"},
+            {"int64", "i64"},
+            {"uint8", "u8"}
+        };
+        auto mapped_type = type_map.find(type_name);
+        if (mapped_type != type_map.end()) {
+            type_name = mapped_type->second;
+        }
+        
+        return Napi::String::New(info.Env(), type_name); 
+    } catch (const std::exception& e) {
+        reportError(info.Env(), e.what());
+        return info.Env().Undefined();
+    }
+}
```

Comment:
Fix last lines of file

Reply:
Done

Reply:
There is still a warning ;)
Please check your editor's settings